src/main/java/app/service/impl/FruitParserImplementation.java

```diff
@@ -0,0 +1,25 @@
+package app.service.impl;
+
+import app.model.SupplyFruit;
+import app.service.FruitParser;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+public class FruitParserImplementation implements FruitParser {
+    public static final int FIRST_ELEMENT = 1;
```

Comment:
```suggestion
    public static final int TYPE_INDEX = 1;
    public static final int FRUIT_NAME_INDEX = 1;
    etc
```
first element is not informative